packages/api/src/handlers/queues.ts

```diff
@@ -95,6 +96,7 @@ async function getAppQueues(
         pagination,
         readOnlyMode: queue.readOnlyMode,
         allowRetries: queue.allowRetries,
+        allowCompletedRetries: queue instanceof BullMQAdapter,
```

Comment:
You don't need to check the type of the adapter, this field can be part of a specific adapter.
maybe put a default value on the Base class & override it when needed.